webvtt_test.go

```diff
@@ -158,8 +158,8 @@ func TestWebVTTEscape(t *testing.T) {
 	require.NoError(t, err)
```

Comment:
Could you add a `&nbsp;` above to test this case as well?

Reply:
Test case removed in favor of test in subtitles.go